### ISSUE TITLE
Missing Namespace error when trying to retrieve tags

### ISSUE THREAD

**Issue by krlohnes** [durch/rust-s3#187]
**Describe the bug**
A panic happens when trying to retrieve tags from minio

```
thread 'name' panicked at 'called `Result::unwrap()` on an `Err` value: MissingNamespace', /usr/local/cargo/git/checkouts/rust-s3-e07d9ff6dd581ded/e0421a1/s3/src/bucket.rs:1166:58
```

Here's [a link to the line number](https://github.com/krlohnes/rust-s3/blob/add-key-value-tag-methods/s3/src/bucket.rs#L1166)
**To Reproduce**
Upload something to minio with a tag and try to retrieve the tag.

**Expected behavior**
The tag retrieval shouldn't panic

**Environment**
 - Rust version: `1.49`
 - lib version: `0.27.0-rc3`

**Additional context**

**Comment by durch** [comment]
@krlohnes thanks for raising this, I expect this happened when the object had not tags, should be fixed in `0.27.0-rc4`

**Comment by durch** [comment]
Closing, please reopen if the issue is still present
